typespec-extension/src/code-model-builder.ts

```diff
@@ -663,9 +663,9 @@ export class CodeModelBuilder {
       // check if the convenience API need to be disabled for some special cases
       if (operationIsJsonMergePatch(op)) {
         // do not generate convenience method for JSON Merge Patch
-        generateConvenienceApi = false;
-        apiComment = `Convenience API is not generated, as operation '${op.operation.name}' is 'application/merge-patch+json'`;
-        this.logWarning(apiComment);
+        // generateConvenienceApi = false;
+        // apiComment = `Convenience API is not generated, as operation '${op.operation.name}' is 'application/merge-patch+json'`;
+        // this.logWarning(apiComment);
```

Comment:
Why are these lines commented out?

Reply:
This is to **turn off disabling convenient method for patch in emitter**, so that models will be generated. Otherwise, if convenient methods are diabled, models will not be generated.

Reply:
You now can just delete these code now.